src/telliot_feed_examples/sources/price/spot/nomics.py

```diff
@@ -52,14 +53,14 @@ async def get_price(self, asset: str, currency: str) -> OptionalDataPoint[float]
 
         url_params = urlencode(
             {
-                "key": API_KEY,
+                "key": "561128ff5c392739fc12577dc1c4e63ccd3991fc",
```

Comment:
Guessing this was an oops. Who's key is this? We probably need to change it, as this is a paid API, right?